File: Problem_9/problem9.py
c = 1
sol = None
while not sol:
    for b in range(1,c):
        for a in range(1,b):
            if a+b+c == 1000 and a*a+b*b == c*c:
                sol = (a,b,c)
    c += 1

print(sol[0]*sol[1]*sol[2])

File: Problem_9/problem9.js
let c = 1
let sol;

while (!sol) {
    for (let b=1; b < c; b++) {
        for (let a=1; a < b; a++) {
            if (a+b+c===1000 && a*a+b*b === c*c)
                sol = [a,b,c]
        }
    }
    c++
}

console.log(sol.reduce((x, y) => x*y))